src/test/java/com/proofpoint/discovery/TestDynamicAnnouncementResource.java

```diff
@@ -23,23 +23,31 @@
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
+import javax.servlet.http.HttpServletRequest;
 import javax.ws.rs.core.Response;
 
 import static javax.ws.rs.core.Response.Status.ACCEPTED;
 import static javax.ws.rs.core.Response.Status.BAD_REQUEST;
 import static javax.ws.rs.core.Response.Status.FORBIDDEN;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
 public class TestDynamicAnnouncementResource
 {
     private InMemoryDynamicStore store;
+    private AuthManager authManager;
     private DynamicAnnouncementResource resource;
+    private HttpServletRequest servletRequest;
 
     @BeforeMethod
     public void setup()
     {
         store = new InMemoryDynamicStore(new DiscoveryConfig(), new RealTimeSupplier());
-        resource = new DynamicAnnouncementResource(store, new NodeInfo("testing"), new DiscoveryConfig().setGeneralPoolMapTarget("SNV"));
+        authManager = new AllowAllAuthManager();
```

Comment:
Shouldn't there be tests that `DynamicAnnouoncementResource` calls its `AuthManager` and doesn't modify anything when that throws an exception?